index.html

```diff
@@ -184,14 +184,16 @@ <h3>bthome-weather-station</h3>
           </div>
         </div>
 
-        <h3 id="example-data">Example Data</h3>
+        <h3 id="ble-advertising">BLE advertising in the BTHome V2 format</h3>
```

Comment:
Given that this page is about the V2 format, should we reference it in the header/text?

Reply:
Added V2 in the first title and the text below.